app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ArticlesController < ApplicationController
+  def index
+    @articles = Article.published.default_order.page(params[:page])
+  end
+
+  def show
+    @article = Article.find(params[:id])
```

Comment:
```suggestion
    @article = Article.published.find(params[:id])
```
つけたい
非公開記事見えちゃうのでは？